src/logic/operate.js

```diff
@@ -0,0 +1,33 @@
+import Big from 'big.js';
+
+const Operate = (numberOne, numberTwo, operation) => {
+  const numO = Big(numberOne);
+  const numT = Big(numberTwo);
+  let result = null;
+  switch (operation) {
+    case '÷': {
+      if (numberTwo === 0) {
+        result = 'Cannot divide by zero';
+      }
+      result = numO.div(numT);
+      break;
+    }
+    case 'X': {
+      result = numO.mul(numT);
+      break;
+    }
+    case '-': {
+      result = numO.minus(numT);
+      break;
+    }
+    case '+': {
+      result = numO.plus(numT);
+      break;
+    }
+    default:
+      break;
+  }
```

Comment:
- Add implementation for the `modifier`. `operate` switches operation and uses `plus`, `minus`, `times`, `div` and `modifier` (turning the number into a percentage based number, e.g. 95 % == .95) functions (screenshot)
```javascript
case operate == '%':
  result = one.mod(two).toString();
  break;
```